webroot/results/includes/_database.php

```diff
@@ -166,7 +166,7 @@ function getAllUsedCountriesCompetitions () {
 #----------------------------------------------------------------------
 function getAllUsedContinents () {
 #----------------------------------------------------------------------
-  return dbQuery("SELECT * FROM Continents");
+  return dbQuery("SELECT * FROM Continents WHERE recordName <> ''");
```

Comment:
I think that join on `Records` actually predates our fake countries. I think it's there just to reduce the set of countries to countries that have actually had someone compete. A side effect of that is that it does filter out the fake multiple countries.

I also don't know how expensive doing a query over the Results table would be... since it works for countries, it's presumably fine to do here. I've implemented it as you suggest over in 20222c80648de5f424385946fdf89d7a97cd53c9.